### PR TITLE
UGLILY FIX the member collections

### PR DESCRIPTION
**Pull request by mixcas**
Ok, before i has changed the logic of the collection/catalogos and stuff. All because client wanted the catalog to be empty again, something that was not considered in the design, neither in the logic. related commits: 
3e7412369439815da51e7d651d878e9c2bc77ec7
05d9d4fc6112a0fee559fece28551354bced1ef5

That really fucked the whole logic of other parts of the app. So I decided to *UGLILY* hardcode the empty catalog and wishlist for now. Rever the logic and restore the broken parts instead of trying to rewrite a whole new logic. 

Also fixed a backbutton issue.

### CODE REVIEW

File: src/native/components/wishlist/WishlistList.js [hunk 1]
```diff
@@ -71,49 +72,46 @@ const WishlistList = ({
             }
           />
         </ScrollView>
+        <Confetti />
```
Comment (mikeyrayvon) [inline src/native/components/wishlist/WishlistList.js]: I think this isn't supposed to be here

Comment (mikeyrayvon) [reply src/native/components/wishlist/WishlistList.js]: Or is this going back in?

**Review by mikeyrayvon — COMMENTED** [review]
Is the plan to revert this merge later, and work out a better strategy?

We can merge this just to appease client, and later start development from before this merge.

File: src/native/components/wishlist/WishlistList.js (hunk 1 above)
Comment (mixcas) [reply src/native/components/wishlist/WishlistList.js]: My bad! Will remove.

**Comment by mixcas** [comment]
@mikeyrayvon the plan is to work out a better strategy for next phase.

**Comment by mikeyrayvon** [comment]
`Directorio > Miembro > Ver Bio (on collection Lote artist) > Back nav` directs to Catálogo, and then there isn't away to get back to main Directorio screen

**Comment by mixcas** [comment]
But that's a different bug

**Comment by mixcas** [comment]
Well, it's a diff bug but it's going to become annoying… sigh. Will fix here

**Comment by mikeyrayvon** [comment]
Your fix breaks navigation from Miembro profile to Directorio, after navigating back from Obra or Bio

**Comment by mikeyrayvon** [comment]
i think this needs an `backToDirectorio` conditional Action

**Comment by mixcas** [comment]
this is good to go

**Pull request merged**